packages/gallery/src/components/gallery/proGallery/slideshowView.js

```diff
@@ -842,15 +842,68 @@ class SlideshowView extends GalleryComponent {
     return <GalleryDebugMessage {...this.props.debug} />;
   }
 
+  getArrowsRenderData() {
+    const { customNavArrowsRenderer, arrowsSize } = this.props;
+    if (customNavArrowsRenderer) {
+      return {
+        arrowRenderer: customNavArrowsRenderer,
+        navArrowsContainerWidth: arrowsSize,
+        navArrowsContainerHeight: arrowsSize
+      }
+    }
+
+    const arrowOrigWidth = 23; //arrow-right svg and arrow-left svg width
+    const arrowOrigHeight = 39; //arrow-right svg and arrow-left svg height
+    const scalePercentage = arrowsSize / arrowOrigWidth;
+    const svgStyle = { transform: `scale(${scalePercentage})` };
+
+    const navArrowsContainerWidth = arrowsSize; // === arrowOrigWidth * scalePercentage
+    const navArrowsContainerHeight = arrowOrigHeight * scalePercentage;
+
+    const svgInternalStyle = {};
```

Comment:
Altough javascript allows a mutation of a constant object, I prefer to avoid it as well (since it's a pitfall of the language IMO). you can do something like this instead:
```
const svgInternalStyle = utils.isMobile() && this.?props?arrowsColor? :  {fill: arrowsColor.value} : {}
```
This way you won't mutate the `svgInternalStyle` (even tough JS allows it)

Reply:
Cool, will do. it was just an old code I moved here.